### ISSUE TITLE
Custom element doesn't exist: fold-entity-row.

### ISSUE THREAD

**Issue by danodemano** [thomasloven/lovelace-fold-entity-row#36]
Brand new install on 0.88 - element not found.  Config seems good though I wasn't sure what type the card was (js or module) but both fail anyway:

```
  - type: module
    url: /local/custom-lovelace/card-tools.js?v=3
  - type: js
    url: /local/custom-lovelace/fold-entity-row.js?v=2
```

Am I missing something stupid?

![image](https://user-images.githubusercontent.com/1128744/53306313-a3a11b00-3859-11e9-8224-434317c53ab3.png)

I also see this in the console of Chrome:

![2019-02-24 17_35_45-desktop](https://user-images.githubusercontent.com/1128744/53306415-d26bc100-385a-11e9-98bb-6305dee4e8e1.png)

**Comment by danodemano** [comment]
I'm an idiot....ignore me

**Comment by poida97** [comment]
what was the problem? turns out I'm an idiot too.

**Comment by poida97** [comment]
got it - I had installed fold-entity-row using HACS. I think that's why I needed to specify a different `resources:` path
`/local/community/lovelace-fold-entity-row/fold-entity-row.js`
instead of
`/local/fold-entity-row.js`

**Comment by Tahutipai** [comment]
Where exactly does one specify that resources path?

**Comment by thomasloven** [comment]
In the place explained in the guide linked from the instructions.

**Comment by Trolann** [comment]
Came here with a fresh install as a dependency of keymaster. Had the same error (doesn't exist) and checked everything listed and your other post about checking via the UI and confirming it was listed as a JavaScript Module (it was). Last ditch effort I set as a Stylesheet to see if it would work and it didn't, so I changed it back to JavaScript Module and the cards began working.

Hassio 2021.8.4 and HACS 1.12.4

**Comment by kevincw01** [comment]
I too came here to get these modules in support of keymaster.  @trolann 's solution of changing the [dashboard module](https://my.home-assistant.io/redirect/lovelace_resources) to stylesheet and back did not work for me.  I had used HACS to install these modules and in this dashboard it had set the location of the javascript files to /hacs/module/module.js?blah...   I'm not sure why /hacs is specified by HACS because my HA doesn't serve files to that location.  I did find out that the correct URL is /local/community/modulename/module.js.  When I changed /hacs to /local/community for each one then they worked.
